src/collections/_documentation/performance/performance-tab.md

```diff
@@ -0,0 +1,46 @@
+---
+title: 'Performance Tab'
+sidebar_order: 1
+---
+
+[SCREENSHOT of Performance Homepage]
+
+The Performance Homepage offers Apdex and Throughput graphs as well as a list of the most problematic transactions so users can see a high-level report of their overall application health.
+
+## Apdex & Throughput Graphs
+The Apdex graph shows the ratio of response times in a user's application against a threshold. Users can get, on aggregate, a sense of how long a customer has waited. For example, if there is a significant dip in an Apdex score in a given window, a user may want to dig into that time period to investigate a potential performance bottleneck.
+
+The Throughput graph indicates the number of transactions over a given time range. Users can quickly visualize the traffic patterns in their application and see if a spike in throughput might be correlated with a dip in their Apdex score.
+
+## Transaction Table
+Underneath the graphs, users can view a list of their transactions with the capability to filter down on which transactions users want to view. In this table, users can sort on the provided columns to understand key characteristics of their transactions like [P95]({%- link _documentation/performance/performance-metrics.md -%}#p95-threshold) duration time, [Average Transaction Duration]({%- link _documentation/performance/performance-metrics.md -%}#average-transaction-duration) time, or [Throughput]({%- link _documentation/performance/performance-metrics.md -%}#throughput).
+
+## My Key Transactions
+"My Key Transactions" are transactions that users bookmark and can view in a separate dedicated tab. Users may want to bookmark a Transaction as a "Key Transaction" if they monitor performance on a particular transaction or group of transactions. Users can set a Transaction as a "Key Transaction" on the Transaction Summary page. The "All Transactions" view also displays these transactions.
+
+## Transaction Summary
+
+[ SCREENSHOT of Transaction Summary page ]
+
+Navigate to the Transaction Summary page by clicking on any Transaction on the Performance Homepage. On the Transaction Summary page, users can find graphs, a list of slowest transactions, facet maps, and summary statistics about the transaction.
```

Comment:
"clicking on any transaction..." (unless we require upper case for this noun when it isn't titling a page)